lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -234,15 +244,18 @@ getTransaction = label "getTransaction" $ do
 
     getTokenTransfer :: Get ([(TxIn, Coin)], [TxOut])
     getTokenTransfer = label "getTokenTransfer" $ do
-        inCount <- fromIntegral <$> getWord8
-        outCount <- fromIntegral <$> getWord8
+        inCount <- fromEnum <$> getWord8
+        outCount <- fromEnum <$> getWord8
         ins <- replicateM inCount getInput
         outs <- replicateM outCount getOutput
         return (ins, outs)
       where
+        convert :: Word8 -> Word32
+        convert = fromIntegral
```

Comment:
```suggestion
    word32 :: a -> Word32
    word32 = fromIntegral
```

Reply:
`Word8 -> Word32` won't overflow. 
`a -> Word32` might overflow depending on the `a`.

I wanted to make this obvious.

Reply:
Removed because of change in overall approach https://github.com/input-output-hk/cardano-wallet/pull/487#issuecomment-507308556